engine/api/src/main/java/io/deephaven/engine/table/Table.java

```diff
@@ -342,8 +381,14 @@ public interface Table extends
     Table formatColumnWhere(String columnName, String condition, String formula);
 
     /**
-     * Produce a new table with the specified columns moved to the leftmost position. Columns can be renamed with the
+     * Produce a new table with the specified columns moved to the rightmost position. Columns can be renamed with the
```

Comment:
Should this say leftmost?
```suggestion
     * Produce a new table with the specified columns moved to the leftmost position. Columns can be renamed with the
```

Reply:
Check the pydocs as well.